### ISSUE TITLE
Using TensorBoard to plot loss and accuracy

### ISSUE THREAD

**Issue by santhoshav15** [pierluigiferrari/ssd_keras#199]
Trying to visualize using TensorBoard in Keras, I added the following few lines for code while training SSD. Searched the web for plotting loss and accuracy in keras using tensorboard, think i am missing out something. Could you guide me here? I'm new to this

`from keras.callbacks import TensorBoard`

`model.compile(optimizer=adam, loss=ssd_loss.compute_loss,metrics=['accuracy'])`

`tensorboard = TensorBoard(log_dir='./logs', histogram_freq=0,
                          write_graph=True, write_images=False,
                          update_freq='epoch')`

`callbacks = [model_checkpoint,
             csv_logger,
             learning_rate_scheduler,
             terminate_on_nan,tensorboard]`

I ran this in the terminal
> tensorboard  --logdir ./logs --port=8008 

I get this while training
![screenshot from 2018-11-28 07-43-20](https://user-images.githubusercontent.com/45348951/49124174-6a502200-f2e1-11e8-9a06-26ca53577c15.png)

OS: Ubuntu 16.04
Keras: 2.2.4
Tensorflow: 1.12.0

**Comment by Shawn0Hsu** [comment]
@santhoshav15 Hi,Have you solved this problem about drawing the loss and accuracy with TensorBoard? I have encountered some errors，could you please give me the correct code about drawing the loss and accuracy with TensorBoard. Thank you !
